cms/djangoapps/contentstore/management/commands/reindex_library.py

```diff
@@ -66,4 +61,5 @@ def handle(self, *args, **options):
             library_keys = map(self._parse_library_key, args)
 
         for library_key in library_keys:
+            print "Indexing library {}".format(library_key)
```

Comment:
Did you mean to leave this in?  If so, should probably import the print function from `__future__` to prepare for Python 3.